bakery-js/package.json

```diff
@@ -15,7 +15,8 @@
     "lint:fix": "prettier --write .",
     "clean": "git clean -dfqX",
     "test": "jest --coverage",
-    "test:ci": "jest --ci --maxWorkers=2 --reporters=default --reporters=jest-junit"
+    "test:ci": "jest --ci --maxWorkers=2 --reporters=default --reporters=jest-junit",
+    "watch": "npx nodemon --inspect=0.0.0.0:9229 --watch dist/index.js "
```

Comment:
Is this called by anything? I didn't find a reference to it anywhere. 

I haven't used nodemon in a long time but does it provide anything more than `node --inspect=0.0.0.0:9229`?

If it is being used for development then it might be good to include nodemon in the list of devDependencies.